docroot/sites/all/themes/ilr_theme/scss/layout/_ilr-at-70-homepage-layout.scss

```diff
@@ -52,7 +51,7 @@ body.subsite-front {
           @include grid-span(7,6);
         }
         ul {
-          left: 42%;
```

Comment:
Is there a better approach than absolutely positioning the share icons and having fragile declarations like these?